.travis.yml

```diff
@@ -4,14 +4,20 @@ dist: trusty
 language: node_js
 node_js:
   - "9.5"
-  
+
 before_script:
   - npm i npm@latest -g
   - npm install -g --silent @angular/cli
 
 script:
   - ng lint
-  - ng build --prod
-
-after_success:
+  - ng build --prod --base-href="/web-app/"
```

Comment:
You might want to set --base-hre="https://openmf.github.io/web-app/" 
It won't work otherwise once you push to gh-pages

Reply:
No I think its not necessary because I tried this on one of my repositories and it worked fine.